### ISSUE TITLE
I upload only the edits I made

### ISSUE THREAD

**Issue by park7231654** [dongseokYang/keras_shufflenetv2#1]
file name : train.py

def training_pipeline_ShuffleNetV2():

    data_dir = 'insect_30'
    image_size = 299

    data_generator = ImageDataGenerator(rescale=1./255, validation_split=0.1)

    train_generator = data_generator.flow_from_directory(
        str(data_dir),
        subset='training',
        target_size=(image_size, image_size),
        batch_size=model_config._BATCH_SIZE,
        class_mode='categorical',
        shuffle=True,
        seed=0)

    validation_generator = data_generator.flow_from_directory(
        str(data_dir),
        subset='validation',
        target_size=(image_size, image_size),
        batch_size=model_config._BATCH_SIZE,
        class_mode='categorical',
        shuffle=True,
        seed=0)

    model = ShuffleNetV2Model()  # your keras model here
    model.compile(optimizer='adam', loss='categorical_crossentropy', metrics=['accuracy'])

    model.fit_generator(
        train_generator,
        steps_per_epoch=?,
        epochs=model_config._EPOCHS,
        validation_data=validation_generator,
        validation_steps=?)

    # display model config
    model.summary()
----------------------------------------------------------------------------------------------------------------------------------
if __name__ == '__main__':
    
    training_pipeline_ShuffleNetV2()
